Improve getting started advice in README (#872)
rust-lang/team: README.md

cargo run check
```

> Note that, if you're using this for the first time, some of these checks will fail due to missing API tokens.

Author: Some of the checks are skipped, producing warnings and the tool otherwise works. Let me amend the wording.

> Note that, if you're using this for the first time, some of these checks will fail due to missing API tokens.

Author: ```suggestion
Note that, if you're using this for the first time, some of these checks will be skipped due to missing API tokens.
```

> Note that, if you're using this for the first time, some of these checks will be skipped due to missing API tokens.

Reviewer: I'm not sure it makes sense to qualify this with "first time". After all, if they run again (or a hundred more times), the tool will still skip some checks. 

```suggestion
Note that some of these checks will be skipped due to missing API tokens.
```

### Adding a person to the repository

It's possible to fetch the public information present in a GitHub profile and
store it in a person's TOML file:

```
cargo run add-person <github-username>
```

> You can also add additional information, such as someone's DIscord or Zulip ID by adding additional fields to their `.toml` file.

Reviewer: ```suggestion
You can also add additional information, such as someone's Discord or Zulip ID by adding additional fields to their `.toml` file.
```
